pom.xml

```diff
@@ -129,10 +127,10 @@
             <layout>default</layout>
             <name>OSS Sonatype Snapshots</name>
             <releases>
-                <enabled>false</enabled>
+                <enabled>true</enabled>
             </releases>
             <snapshots>
-                <enabled>true</enabled>
+                <enabled>false</enabled>
```

Comment:
It must be `true`, official release are automatically fetched from the Maven Central